recipes/pulseaudio/all/conanfile.py

```diff
@@ -0,0 +1,137 @@
+from conans import ConanFile, tools, AutoToolsBuildEnvironment, RunEnvironment
+from conans.errors import ConanInvalidConfiguration
+import os
+import glob
+import shutil
+
+
+class LibnameConan(ConanFile):
+    name = "pulseaudio"
+    description = "PulseAudio is a sound system for POSIX OSes, meaning that it is a proxy for sound applications."
+    topics = ("conan", "pulseaudio", "sound")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "http://pulseaudio.org/"
+    license = "LGPL-2.1"
+
+    generators = 'pkg_config'
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "with_alsa": [True, False],
+        "with_glib": [True, False],
+        "with_fftw": [True, False],
+        "with_x11": [True, False],
+        "with_openssl": [True, False],
+        "with_dbus": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "with_alsa": True,
+        "with_glib": False,
+        "with_fftw": True,
+        "with_x11": True,
+        "with_openssl": True,
+        "with_dbus": True,
+    }
+
+    build_requires = "gettext/0.20.1", "libtool/2.4.6"
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    _autotools = None
+
+    def configure(self):
+        if self.settings.os != "Linux":
+            raise ConanInvalidConfiguration("pulseaudio supports only linux currently")
+        del self.settings.compiler.libcxx
+        del self.settings.compiler.cppstd
+
+
+    def requirements(self):
+        self.requires("libsndfile/1.0.28")
+        if self.options.with_alsa:
+            self.requires("libalsa/1.1.9")
+        if self.options.with_glib:
+            self.requires("glib/2.64.0")
+        if self.options.with_fftw:
+            self.requires("fftw/3.3.8")
+        if self.options.with_x11:
+            self.requires("xorg/system")
+        if self.options.with_openssl:
+            self.requires("openssl/1.1.1g")   
+
+    def system_requirements(self):
+        installer = tools.SystemPackageTool()
+        if tools.os_info.with_apt:
+            installer.install('libcap-dev')
```

Comment:
TODO

Reply:
It deserves a proxy package. libcap can be found under Linux repo: https://git.kernel.org/pub/scm/libs/libcap/libcap.git/refs/ I don't know how many dependencies it requires to be built.

Reply:
Libcap is a userland library and should get a regular package. It even has implementations in Go AFAIK

Someone started to work on it:  #3397